### ISSUE TITLE
Shared dashboard links

### ISSUE THREAD

**Issue by Anaisdg** [influxdata/ui#4662]
**Proposal:**
The ability to share dashboards like you can share notebooks. 

**Current behavior:**
Feature is only available for notebooks. 

**Use case:**
Allow read only users or people outside of organizations to view data from InfluxDB Dashboards.

**Comment by asalem1** [comment]
Not sure how big a priority this is given the large infrastructural changes that would be required for making it possible. In the mean time, if there's a dashboard you've got that you'd like to share, it can be converted to a notebook (given that it doesn't use variables). @Anaisdg do you have a link to the dashboard? I can help out with converting it to a notebook to be shared

**Comment by Iotnet** [comment]
There is an update on that feature  ?

**Comment by xiaoshenxian** [comment]
Yeah, this feature is eagerly required. It should be at least to allow to create a readonly account to login the UI.

**Comment by modem-man-gmx** [comment]
Any way of doing this? It would very much help here, in my non-profit / non-government environmental health organisation, to make some of my dashboards publicly accessible by citizens as well as by politicians.

@Anaisdg, if there are "other ways", like setting up a docker container which somehow magically embeds a single dash's graphic at medium refresh rate (or the like) please give me a hint. I actually have no clue how I could set up such access from a, say, nginx container in detail.

Another proposal would be implementing a pure anonymous, read-only exposure of (one/some/all) dashes at a second web interface. Should run as a second thread or process, offering a different port (defaulting to :443). This could get configured entirely at the :8086 pages and using no interactive input values at all, so no infrastructural changes.

@asalem1, you noted "large infrastructural changes". Please think about a bypass/sidewalk solution, like mentioned above. Could become a separate github project with only a small initial amount of your support required. From your answer I assume you would have the deep insight to Influx and APIs, to push the cariot in the right rails...
